<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>墨色预制计文件传输程序</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>本地IP地址:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>远程IP地址:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文件存放目录</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_localAddress">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_remoteAddress">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_filePath">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>60</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_localPort">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_search</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>findFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findFile()</slot>
  <slot>sendFile()</slot>
  <slot>cancelTransmission()</slot>
 </slots>
</ui>
